_posts/en/newsletters/2020-06-03-newsletter.md

```diff
@@ -0,0 +1,299 @@
+  focused on use cases and the benefits of using LSAT, such as enhanced
+  user privacy and improved sign up experience.
+  ([transcript](https://diyhpl.us/wiki/transcripts/vr-bitcoin/2020-05-16-oliver-gugger-lsat/),
+  [video](https://www.youtube.com/watch?v=IW08RJUpzw0))
+
+- **Sydney meetup discussion:** A number of Bitcoin and LN developers
+  joined this Sydney meetup to discuss topics including: the scalability
+  issues of onboarding millions of LN clients, Rust code integration
+  into Bitcoin Core, dual funding in C-Lightning, and modern soft fork
+  activation. The history of Linux kernel development and segwit
+  activation were explored for insights into when or whether the Rust
+  language should be introduced to Bitcoin Core and the exact mechanism
+  that should be used to activate proposed soft forks such as
+  [taproot][topic taproot] in future. The transcript was anonymized to
+  encourage open discussion.  ([transcript][sydney xs])
+
+- **Revault—A Multiparty Vault Architecture:** Kevin Loaec and Antoine
+  Poinsot presented on their vault design *Revault* at London Bitcoin
+  Devs. They outlined specific details such as its reliance on
+  co-signing servers and how it compares to some other vault designs
+  that require key deletion, anticipating spending amounts, or both.
+  Their presentation was preceded the week before by a wider discussion
+  on vaults, [covenants][topic covenants], and `OP_CHECKTEMPLATEVERIFY`
+  (`OP_CTV`). This included the merits of the different use cases of
+  `OP_CTV` and a possible path to it being soft forked into Bitcoin.
+  Additional discussion focused on the current state of the the mempool
+  policy in Bitcoin Core and how it creates challenges such as
+  [transaction pinning][topic transaction pinning] that affect the
+  security of both vault designs and the LN protocol.  ([Meetup
+  transcript][london xs], [Presentation transcript][revault xs], [Meetup
+  video][london vid], [Presentation video][revault vid])
+
+## Releases and release candidates
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+- [Bitcoin Core 0.20.0rc2][bitcoin core 0.20.0] is the most recent
+  release candidate for the next major version of Bitcoin Core.
+
+- [LND 0.10.1-beta.rc3][] is the latest release candidate for the next
+  maintenance release of LND.
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo],
+[C-Lightning][c-lightning repo], [Eclair][eclair repo], [LND][lnd repo],
+[Rust-Lightning][rust-lightning repo], [libsecp256k1][libsecp256k1 repo],
+[Bitcoin Improvement Proposals (BIPs)][bips repo], and [Lightning
+BOLTs][bolts repo].*
+
+*Note: the commits to Bitcoin Core mentioned below apply to its master
+development branch and so those changes will likely not be released
```

Comment:
I think "so those changes" can be omitted (or replaced by ", and unless backported,")

Reply:
Backporting is one concern, but my main concern is promising that X merged feature will be in release Y.  There's always the chance that something will be reverted before then.  I agree that this warning is probably more verbose than necessary given that all reasonable developers understand that things can change between a development merge and a release---but a few years ago I polled some Bitcoin developers for some deadlines on segwit stuff, used that to make a table of planned release dates, and then saw /r/btc republish that table over and over again in attacks on the project when those dates weren't met.  Several core devs have told me they don't blame me for what happened, but ever since I've been *very* careful about writing anything that sounds like a promise to release specific features, much less promises to release at a certain point in time.